src/components/Onboarding/Screens/Apps/AgreementSettings/index.js

```diff
@@ -222,8 +222,8 @@ function AgreementSettings() {
         >
           <span>
             Drag and drop a document here or <TextFileUploader /> to upload your
-            community covenant. By uploading this file, you agree to it being
-            uploaded to IPFS.
+            community covenant. By uploading a file, you agree to Gardens
+            uploading this file to IPFS.
```

Comment:
should be changing this text here?

Reply:
It wasn't me, this came from other commit.